server/sonar-server-common/src/main/java/org/sonar/server/user/index/UserIndexer.java

```diff
@@ -93,14 +92,10 @@ public void commitAndIndex(DbSession dbSession, Collection<UserDto> users) {
 
     dbClient.esQueueDao().insert(dbSession, items);
     dbSession.commit();
-    postCommit(dbSession, users.stream().map(UserDto::getLogin).collect(toList()), items);
+    postCommit(dbSession, items);
   }
 
-  /**
-   * Entry point for Byteman tests. See directory tests/resilience.
-   * The parameter "logins" is used only by the Byteman script.
-   */
-  private void postCommit(DbSession dbSession, Collection<String> logins, Collection<EsQueueDto> items) {
+  private void postCommit(DbSession dbSession, Collection<EsQueueDto> items) {
```

Comment:
As indicated by the comment, the `users` parameter should not be removed, as it's used by Byteman to change the indexation behaviour at runtime during tests.